decidim-proposals/lib/decidim/proposals/commentable_proposal.rb

```diff
@@ -36,7 +36,13 @@ def users_to_notify_on_comment_created
         end
 
         def user_allowed_to_comment?(user)
-          can_participate_in_space?(user)
+          return unless can_participate_in_space?(user)
+
+          user_authorized_to_comment?(user)
+        end
+
+        def user_authorized_to_comment?(user)
+          ActionAuthorizer.new(user, "comment", component, self).authorize.ok?
```

Comment:
```suggestion
          user_authorized_to? "comment", user
        end

        # this method could go in the generic `commentable.rb`
        def user_authorized_to?(action, user)
          ActionAuthorizer.new(user, "comment", component, self).authorize.ok?
```
I suggest to move the authorized_to_comment to the commentable so it can be reused for other components using comments permissions.